server/templates/base_page.html

```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script async>
```

Comment:
Grooossss. Okay, well, update the comment in that file adding this additional step to deal with if we want to re-enable minification.